### ISSUE TITLE
LWJGL 3.x support

### ISSUE THREAD

**Issue by shadowislord** [jMonkeyEngine/jmonkeyengine#314]
Need to add support for LWJGL 3.x.

There shouldn't be any issues, except the loss of LwjglCanvas which may break the SDK.
AWT panels need to be stable and working before we can fully make the switch.

**Comment by DannyJo** [comment]
I'm currently looking at this and the conclusion is to create a new module (jme3-lwjgl3) which can be dropped in place of the jme3-lwjgl module in order to use the lwjgl 3 renderer.

So far everything is going OK, I've got the GLFW window running and keyboard and mouse input are working as expected. Joysticks still need some work. Audio seems to be fine as well.

I also need to check that the offscreen rendering is working and test the SDK with it to ensure the canvas rendering is OK.

**Comment by normen** [comment]
The SDK already uses AWT panels by default, canvas is optional.

**Comment by DannyJo** [comment]
This work has now been merged into master as of pull request #340
